pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1320,6 +1323,10 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			if ifaceType == "virtio" && virtioNetProhibited {
 				return fmt.Errorf("In-kernel virtio-net device emulation '/dev/vhost-net' not present")
 			} else if ifaceType == "virtio" && virtioNetMQRequested {
+				if *numQueues > multiQueueMaxQueues {
```

Comment:
I think that we now need to encapsulate this calculation again. Now the vCPU count is not always matching the queue count. We probably need a helper which we would call also in virt-handler where we create the tap device.

Reply:
Right. How about now ?